src/services/PhoneHome.php

```diff
@@ -164,4 +164,14 @@ private static function _modules(): string
 
         return implode(PHP_EOL, $modules);
     }
+
+    /**
+     * Check if existing Phoning home job is already in the queue
+     *
+     * @return boolean
+     */
+    private static function _doesQueueJobExist(): bool
+    {
+        return array_search('Phoning home', array_column(Craft::$app->queue->jobInfo, 'description')) !== false;
```

Comment:
ah i guess the key it returns could be falsey, eh

Reply:
Exactly, if it's the first job in the queue, it returns 0